.github/workflows/e2e.yml

```diff
@@ -7,7 +7,11 @@ on:
 jobs:
   playwright:
     name: 'Playwright Tests'
+    timeout-minutes: 60
```

Comment:
This is way to high: a mistake here will cost us 1h of GHA minutes. e2e tests for an app as small as this one should finish in less than 5-10, including the build.